firebase_admin/ml.py

```diff
@@ -246,20 +245,12 @@ def display_name(self, display_name):
     @property
     def create_time(self):
         """The time the model was created."""
-        seconds = self._data.get('createTime', {}).get('seconds')
-        if not isinstance(seconds, numbers.Number):
-            return None
-
-        return datetime.datetime.fromtimestamp(float(seconds))
+        return self._data.get('createTime', None)
```

Comment:
We generally expose timestamps as milliseconds since epoch values in Python. We might want to retain that behavior here as well. String timestamps are kind of awkward in Python (since it requires the user to handle the timezone and the format of the string).

https://github.com/firebase/firebase-admin-python/blob/00cf1d34fdd8ca3531ff3556688c2e785db17920/firebase_admin/_user_mgt.py#L51-L66

What did we agree to in the API review for this?

Reply:
Millis. I changed it to do the conversion. Thanks.